course_catalog/admin.py

```diff
@@ -145,6 +146,15 @@ class PodcastEpisodeAdmin(admin.ModelAdmin):
     search_fields = ("full_description",)
 
 
+class EnrollmentAdmin(admin.ModelAdmin):
+    """Enrollment Admin"""
+
+    model = Enrollment
+    search_fields = ("run_id", "user_id")
+    list_display = ("run_id", "user_id", "enrollment_timestamp")
+    search_fields = ("run_id", "user_id")
```

Comment:
`search_fields` occurs twice.  Depending on whether or not Peter/Brian think they might need to use this, might be worth making it more user-friendly by doing something like this:

```python
    def run_slug(self, obj):
        return f"{obj.run.slug}"

    run_slug.admin_order_field = 'run__slug'

    model = Enrollment
    search_fields = ("run__run_id", "run__slug", "run__title", "user__email", "user__username")
    list_display = ("run_slug", "user", "enrollment_timestamp")
    autocomplete_fields = ("run", "user")
```

Reply:
Thanks! I updated the code to use your suggestion